css-view-transitions-1/Overview.bs

```diff
@@ -371,6 +397,8 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 	This means the snapshot canvas size is likely to be consistent for the [=document element=]'s [=captured element/old image=] and [=captured element/new element=].
 
+	The [=snapshot viewport=] is also a [=pseudo-element parent=].
```

Comment:
I didn't follow this. The snapshot viewport is just a conceptual containing block, not a pseudo-element.

Reply:
Yeah, I was unsure about this too. Any feelings on the right terms to use here? When we add a `::view-transition`, what are we adding it to?